### PR TITLE
[ticket-2] adds baseline index file and preliminary css

### PR DESCRIPTION
**Pull request by justinw248**
Adds landing page and early css.

### CODE REVIEW

File: css/app.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
+* {
+    margin: 0px;
+}
+
+html {
+
+}
```
Comment (zefrof) [inline css/app.css]:
```suggestion
```

File: css/app.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+body {
+    background-color: #ececec;
+    color: #414141;
+}
+
+nav{
```
Comment (zefrof) [inline css/app.css]:
```suggestion
nav {
```

File: index.html [hunk 3]
```diff
@@ -1,7 +1,53 @@
...
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/foundation.min.css">
+    <link rel="stylesheet" href="css/app.css">
+    <!-- Remove next link in final -->
+    <link rel="stylesheet" href="css/testing.css">
+    <!-- Below Calls to Google Fonts API -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&family=Questrial&display=swap" rel="stylesheet">
```
Comment (zefrof) [inline index.html:15]: Should we move some/all of these to the footer?

Comment (justinw248) [reply index.html:15]: Those that we keep should be moved there, yes. I'll adjust before my next commit.

**Pull request merged**